### DISCUSSION TITLE
How to change taskbar color to be dark (i do not want to change my windows to dark mode)

### DISCUSSION THREAD

**Discussion by githubberguy** [valinet/ExplorerPatcher#4075] [Q&A]
so i do not wanna switch to dark mode, settings does NOT let me change it, help!!!!

**Comment by pyrates999** [comment]
https://support.microsoft.com/en-us/windows/how-to-use-the-taskbar-in-windows-0657a50f-0cc7-dbfd-ae6b-05020b195b07#WindowsVersion=Windows_11
